### PR TITLE
Filthy Station Trait

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Trait that makes station filthy, with a chance to cause trash and even mice to spawn.

## Why It's Good For The Game

Maybe janitors have a job to do?

## Changelog
:cl:
add: Filthy station trait.
/:cl:

### CODE REVIEW

File: code/game/turfs/open/dirtystation.dm [hunk 1]
```diff
@@ -33,8 +33,28 @@
 		new /obj/effect/decal/cleanable/dirt(src)	//vanilla, but it works
 		return
 
-
-	if(prob(80))	//mid dirt  - 1/15
+	if (HAS_TRAIT(SSstation, STATION_TRAIT_FILTHY))
+		if (prob(50))
```
Comment (PowerfulBacon) [inline code/game/turfs/open/dirtystation.dm:37]: Scp13 optimisation

**Comment by bloons3** [comment]
Isn’t this already thing with the hangover station trait?

**Comment by eeSPee** [comment]
> Isn’t this already thing with the hangover station trait?

No, hangover spawns beer bottles next to spawned crewies.

This one just makes tiles more dirty and with trash on them.

**Comment by bloons3** [comment]
So what’s the real difference?

**Comment by eeSPee** [comment]
This one makes the tiles dirty and places trash around the station.

File: code/game/turfs/open/dirtystation.dm [hunk 2]
```diff
@@ -33,8 +33,28 @@
...
+			return	
+		else if (prob(30))
+			new /obj/effect/decal/cleanable/dirt(src)	//plain old dirt
+			return
```
Comment (utilbird) [inline code/game/turfs/open/dirtystation.dm:41]: You don't need `else if` if the statement returns after

**Pull request closed without merging**